pkg/kapp/resources/mod_field_remove.go

```diff
@@ -92,6 +92,19 @@ func (t FieldRemoveMod) apply(obj interface{}, path Path) error {
 			default:
 				panic(fmt.Sprintf("Unknown array index: %#v", part.ArrayIndex))
 			}
+		case part.RegexObj.Regex != nil:
```

Comment:
Shouldn't we first check for `part.RegexObj` to be non nil? (Also, same as above, we can check both in `case`)

Reply:
Adopted